app/views/Partners/PartnersOverview.tsx

```diff
@@ -0,0 +1,127 @@
+
+// For fixing image width issues
+const win = Dimensions.get('window');
+
+type PartnersScreenProps = {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+};
+
+const PartnersScreen = ({ navigation }: PartnersScreenProps): JSX.Element => {
+  const { t } = useTranslation();
+  const [toggleActive, setToggleActive] = useState(false); // mocked
+  const navigateToViewHAs = () => navigation.navigate('PartnersEdit');
+
+  return (
+    <NavigationBarWrapper
+      title={t('authorities.title')}
+      includeBackButton={false}
+      onBackPress={() => {}}>
+      <ScrollView
+        style={styles.backgroundWrapper}
+        alwaysBounceVertical={false}
+        contentContainerStyle={{ flexGrow: 1 }}>
+        <Image
+          // Hard code to the dimensions of the image. Fixes ScrollView issues
+          style={{ width: win.width, height: (win.width * 900) / 1125 }}
+          resizeMode={'contain'}
+          source={Images.Doctors}
+        />
+        <View style={{ height: 20 }} />
+        <View style={styles.horizontalPadding}>
+          <Typography use={'headline2'}>{t('authorities.title')}</Typography>
+          <View style={{ height: 8 }} />
+          <Typography use={'body1'}>{t('authorities.info_body')}</Typography>
+        </View>
+        <View style={{ height: 20 }} />
+        {/* Outside horizontal padding so touchable is full width */}
+        <View style={styles.divider} />
+        <TouchableHighlight
+          underlayColor={Colors.UNDERLAY}
+          onPress={navigateToViewHAs}>
+          <View
+            style={[
+              styles.horizontalPadding,
+              {
+                flexDirection: 'row',
+                justifyContent: 'space-between',
+                paddingVertical: 20,
+              },
+            ]}>
+            <Typography use={'body1'} style={{ fontWeight: '500' }}>
+              {t('authorities.view_button_label')}
+            </Typography>
+            <SvgXml xml={Icons.ChevronRight} height={20} />
+          </View>
+        </TouchableHighlight>
+        <View style={[styles.divider, { marginHorizontal: 24 }]} />
+      </ScrollView>
+      <View style={styles.bottomSheet}>
+        <Typography
+          style={{
+            // Prevent from forcing overflow on parent
+            flex: 1,
+          }}>
+          {t('authorities.automatically_follow')}
+        </Typography>
+        <View style={{ width: 16 }} />
+        <Switch
+          trackColor={{ true: '#6979F8', false: '#6979F8' }}
+          value={toggleActive}
+          onValueChange={setToggleActive}
+        />
+      </View>
+    </NavigationBarWrapper>
+  );
+};
+export default PartnersScreen;
+
+const styles = StyleSheet.create({
+  backgroundWrapper: {
+    backgroundColor: Colors.INTRO_WHITE_BG,
+    flex: 1,
+  },
+  divider: {
+    height: StyleSheet.hairlineWidth,
+    width: '100%',
+    backgroundColor: Colors.VIOLET_BUTTON_LIGHT,
```

Comment:
Why are we using button colors for the background color?

Reply:
Just a color match. More motivation to standardize our colors, the current color palettes are outdated

Reply:
as a heads up, this one consolidated the colors constants all into the styles module: https://github.com/Path-Check/covid-safe-paths/pull/1015